examples/ondemand.go

```diff
@@ -35,6 +35,14 @@ func exampleOndemand() {
 	subtitleRep, _ := subtitleAS.AddNewRepresentationSubtitle(256, "captions_en")
 	_ = subtitleRep.SetNewBaseURL("http://example.com/content/sintel/subtitles/subtitles_en.vtt")
 
+	thumbnailsAS, _ := m.AddNewAdaptationSetThumbnails(mpd.DASH_MIME_TYPE_IMAGE_JPEG)
+	_,_ = thumbnailsAS.SetNewSegmentTemplateThumbnails(1801800, "$RepresentationID$/$Number$.jpg", 0, 30000)
+	_,_ = thumbnailsAS.AddNewRepresentationThumbnails(50000, "5x4", 1600, 720, "http://dashif.org/guidelines/thumbnail_tile")
+
 	mpdStr, _ := m.WriteToString()
 	fmt.Println(mpdStr)
 }
+
+func main() {
```

Comment:
I think you can remove this function

Reply:
it was originally in the live file, just moved it here to test the thumbnails code https://github.com/zencoder/go-dash/pull/83/files#diff-97e3253cd66b3f6c0e3c97c23211571eL9, does it make a difference? or should I restore it to how it was originally?

Reply:
Yep, just revert to where it was